.drone.yml

```diff
@@ -14,8 +14,11 @@ notify:
 publish:
   catapult:
     url: $$catapult_url
-    application: "marathon-stats"
+    applications:
+      - "marathon-stats-dev"
```

Comment:
I still don't quite understand when to use dev-infra or dev.  I figured I should use `-dev` to match the name of the cluster it's getting stats from.